docs/references/vod/upload.mdx

```diff
@@ -48,6 +48,7 @@ curl --location --request POST 'https://livepeer.studio/api/asset/request-upload
 ```jsx
 {
     "url": "https://origin.livepeer.studio/api/asset/upload/aHR0cHM6Ly9zdG9yYWdlLmdvb2dsZWFwaXMuY29tL2xwLW55Yy12b2QtbW9uc3Rlci9kaXJlY3RVcGxvYWQvNjI3OHQyZWlrY2JqZXUxNC9zb3VyY2U%2F...",
+    "tusEndpoint: "https://origin.livepeer.studio/api/asset/upload/tus?uploadToken=aHR0cHM6Ly9zdG9yYWdlLmdvb2dsZWFwaXMuY29tL2xwLW55Yy12b2QtbW9uc3Rlci9kaXJlY3RVcGxvYWQvNjI3OHQyZWlrY2JqZXUxNC9zb3VyY2U%2F...",
```

Comment:
```suggestion
    "tusEndpoint": "https://origin.livepeer.studio/api/asset/upload/tus?uploadToken=aHR0cHM6Ly9zdG9yYWdlLmdvb2dsZWFwaXMuY29tL2xwLW55Yy12b2QtbW9uc3Rlci9kaXJlY3RVcGxvYWQvNjI3OHQyZWlrY2JqZXUxNC9zb3VyY2U%2F...",
```

Reply:
Can't believe I missed another one on the fix! thanks